templates/admin/forms/list.html.twig

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style type="text/css">
+        {# Specific override to allow larger width on this container#}
+        .container {
+            width: 1500px;
+        }
+        .bottom-pad {
+            padding-bottom: 25px;
+        }
+    </style>
+    <title>{{ "Forms Administration"|translate }}</title>
+</head>
+<body>
+<div class="container">
+    <form action="./forms_admin.php" method="post">
```

Comment:
need to work in the session fix ( onsubmit='return top.restoreSession() ) :
http://www.open-emr.org/wiki/index.php/OpenEMR_System_Architecture#PHP_Sessions_and_Browser_Windows